example/package.json

```diff
@@ -12,8 +12,8 @@
     "express": "^4.12.3",
     "logfmt": "^1.2.0",
     "node-jsx": "^0.13.3",
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0",
+    "react": "^0.15.0",
+    "react-dom": "^0.15.0",
```

Comment:
Hi @martinandert. I've pushed the fix, sorry for remaining typo in version number.